frontend/package.json

```diff
@@ -8,12 +8,6 @@
     "@material-ui/icons": "^4.9.1",
     "@react-keycloak/web": "2.1.0",
     "@reduxjs/toolkit": "1.2.3",
-    "@types/classnames": "^2.2.10",
-    "@types/crypto-js": "3.1.44",
-    "@types/mui-datatables": "^2.13.7",
-    "@types/react-datepicker": "2.11.0",
-    "@types/react-paginate": "6.2.1",
-    "@types/yup": "0.26.32",
     "ag-grid-community": "^23.0.2",
```

Comment:
the typings belong in the `devDependencies` section of **package.json** - they are not required in the production build.